messaging-experiments.json

```diff
@@ -263,12 +263,12 @@
     }
   },
   {
-    "id": "PROTECTION-RETENTION-EXPERIMENT-RELEASE",
+    "id": "PROTECTION-RETENTION-EXPERIMENT-RELEASE-BUG-1645918",
     "enabled": true,
     "filter_expression": "(env.channel == 'release' || env.channel == 'default') && env.version >= '77.'",
     "targeting": "localeLanguageCode == 'en' && [userId, \"messaging-experiments-cfr\"]|bucketSample(0, 310, 10000) && 'app.shield.optoutstudies.enabled'|preferenceValue",
     "arguments": {
-      "slug": "bug-1645918-high-velocity-message-testing-privacy-retention-study",
+      "slug": "bug-1645918-high-velocity-message-testing-privacy-retention-study-release",
```

Comment:
Hmm, seeing this `bug-1646798-message-high-velocity-message-testing-privacy-retenti-release-77-79` on experimenter